packages/apollo-server-defer/src/execute.ts

```diff
@@ -0,0 +1,1630 @@
+/**
```

Comment:
One thing that makes this file harder to review is that it's hard to know what comes from graphql.js and what is new. Can you point me to some of the most important functions to review?

Reply:
Definitely. Let me strip out all the utility functions and leave only those that changed.